packages/matchbox/src/components/Pager/Pager.js

```diff
@@ -1,21 +1,34 @@
-import React, { Component } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { margin } from 'styled-system';
+import { createPropTypes } from '@styled-system/prop-types';
+import styled from 'styled-components';
+
 import Next from './Next';
 import Previous from './Previous';
-import styles from './Pager.module.scss';
+import { Box } from '../Box';
 
-class Pager extends Component {
-  static displayName = 'Pager';
+const StyledPager = styled(Box)`
```

Comment:
I made this mistake in another ticket too but this actually needs to be `styled('div')`, using box here will actually allow all system props to be used not just margin